__tests__/users.ts

```diff
@@ -320,19 +320,41 @@ describe('query userStreaks', () => {
     }
   }`;
 
+  const QUERY_BY_USER_ID = `query UserStreak($id: ID) {
+    userStreak(id: $id) {
+      max
+      total
+      current
+      lastViewAt
+    }
+  }`;
+
   afterEach(() => {
     jest.useRealTimers();
   });
 
-  it('should not allow unauthenticated users', () =>
-    testQueryErrorCode(client, { query: QUERY }, 'UNAUTHENTICATED'));
+  it('should throw error when no user id provided and no user is authenticated', async () => {
+    await testQueryErrorCode(
+      client,
+      { query: QUERY },
+      'GRAPHQL_VALIDATION_FAILED',
+      'User is missing!',
```

Comment:
Not sure we need this additions as it won't ever persist to the FE

Reply:
The message addition u mean? can remove

Reply:
Yes, don't think it will add much if we just use it for test cases basically.